stack/eslint-config/index.js

```diff
@@ -7,6 +7,8 @@ module.exports = {
   // Disable the parser by default
   parser: '',
 
+  root: true,
```

Comment:
Was this intentional?

Reply:
This option makes eslint ignore personal configuration files like `$HOME/.eslintrc.json` and use only the configuration provided by the `api-extractor` project. Probably you don't have a personal eslint configuration file, and therefore you haven't seen the following deprecation warning.

```
DeprecationWarning: The '~/.eslintrc.*' config file has been deprecated. Please remove it or add 'root:true' into the config file of your projects in order to avoid loading '~/.eslintrc.*' accidentally.
```

Both alternatives are equivalent. Except if you must work on very old legacy projects (bower, grunt, ...) for which eslint is not available, and you have to install it yourself as a global install with personal configuration.

If you think otherwise let me know I will remove it.

Reply:
I just removed this line